worlds/witness/options.py

```diff
@@ -35,12 +35,17 @@ class EarlyCaves(Choice):
     alias_on = 2
 
 
-class EarlySymbolItem(DefaultOnToggle):
+class EarlyGoodItems(OptionSet):
     """
-    Put a random helpful symbol item on an early check, specifically Tutorial Gate Open if it is available early.
+    Put one random helpful item of each of the chosen types on an early check, specifically a sphere 1 Tutorial location.
+
+    If there aren't enough sphere 1 Tutorial locations, Tutorial First Hallway Straight and Tutorial First Hallway Bend may be added as locations.
+    If there still aren't enough sphere 1 Tutorial locations, a random local sphere 1 location is picked.
+    If no local sphere 1 locations are available, there are no further attempts to place the item.
     """
 
-    visibility = Visibility.none
+    valid_keys = {"Symbol", "Door / Door Panel", "Obelisk Key"}
+    default = {"Symbol"}

```

Comment:
The default is a frozenset, so PyCharm/mypy wants that used for these, whether it _should_ 🤷

Reply:
Oh I forgot to run mypy